docs/explanation/parts.rst

```diff
@@ -0,0 +1,263 @@
+Parts
+=====
+
+In the Craft Parts framework, parts are descriptions of the components to be
+build for deployment, either individually or as part of a larger build
+containing many components.
+
+Tools like `Snapcraft`_ and `Juju`_ that use the concepts of parts to
```

Comment:
curiousity: When you say Juju, does this refer to charmcraft being part of Juju?

Reply:
Yes, I think I meant charmcraft. :slightly_smiling_face: